.circleci/package.json

```diff
@@ -9,6 +9,7 @@
   "author": "",
   "license": "ISC",
   "dependencies": {
+    "@orbs-network/judge-dredd": "^0.0.6",
```

Comment:
what's this?

Reply:
This is the external package that marvin-analysis calls - it JUDGES whether the test passed or failed. Its logic is external to this repo because we want to be able to update it separately.